components/ButtonBar/tests/unit/ButtonBarButtonTitleFontTests.swift

```diff
@@ -32,7 +33,7 @@ class ButtonBarButtonTitleFontTests: XCTestCase {
     // Then
     for view in buttonBar.subviews {
       if let button = view as? MDCButton {
-        XCTAssertNil(button.titleFont(for: .normal))
+        XCTAssertTrue(button.titleFont(for: .normal)!.mdc_isSimplyEqual(MDCTypography.buttonFont()))
```

Comment:
_Tip:_ Safely unwrap this in an `if let` and then in the `else` print out a logical error message.

Reply:
Done